example_project_app/plugins.py

```diff
@@ -19,6 +19,21 @@
 )
 
 
+def get_example_setting_default(project=None, user=None):
+    """Example callable function for different scopes
+    :param project: Project object
+    :param user: User object
+    :return: String with project and user info or 'No project'"""
+    response = 'Example callable setting'
```

Comment:
This implementation looks good, but I think we should change the default response to something like "No project or user for callable". This makes it clear in the UI that the callable is working.